rules/downgrade/src/Rector/DowngradeRectorTrait.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Downgrade\Rector;
+
+trait DowngradeRectorTrait
+{
+    /**
+     * Run the rector only when the feature is not supported
+     */
+    abstract protected function getPhpVersionFeature(): string;
+}
```

Comment:
We're trying to avoid traits (except for dependency injection decoupling of `AbstractRector` class), as they're very hard to declutter. 

This should be `interface` instead, wich clear contract.

Reply:
Alright, replaced as `DowngradeRectorInterface`